src/gateway/kong-production/monitoring/statsd.md

```diff
@@ -1,6 +1,99 @@
 ---
-title: Monitoring with Statsd
-
+title: Monitoring with StatsD
+content_type: how-to
 ---
 
-## PLACEHOLDER How to Monitor with Statsd
\ No newline at end of file
+[StatsD](https://github.com/statsd/statsd) is a network daemon that collects
+and aggreates performance metrics by listening on the network for simple 
+text based statistics data published by applications. 
+
+This guide will help you setup a test {{site.base_gateway}} and
+StatsD service. Then you will generate sample requests to {{site.base_gateway}} and
+observe the collected monitoring data. 
+
+### Prerequisites
+* [Docker](https://docs.docker.com/get-docker/) is installed locally. It is used to run {{site.base_gateway}}, the supporting database, and StatsD locally. 
+
+   If you will be using an existing {{site.base_gateway}} instance instead of the test instance, you may need configure Docker to ensure features like network connectivity function correctly.  
```

Comment:
```suggestion
   If you will be using an existing {{site.base_gateway}} instance instead of the test instance, you may need configure Docker to ensure Docker and {{site.base_gateway}} are running on the same network.  
```
@rspurgeon For your comment about UDP, would something like this work?

Reply:
@cloudjumpercat I've just pushed a revision, please take a look.  I think since we're making the Docker based GW optional, we have to be vague about the instructions for the user wishing to use their own.  UDP networking rules are not easily described here, but it won't be accurate to say that the StatsD in Docker and the existing GW have to be on the "same network" because StatsD is ran in a new Docker network.

In fact, I just realized, the StatsD docker run command is not going to work OOTB if the user does not use the script because it uses a new network which won't exist (--network=kong-quickstart-net), so the user will need to navigate this with our intentionally vague instructions here.